fedimg/services/ec2/ec2imgpublisher.py

```diff
@@ -238,7 +238,7 @@ def copy_images_to_regions(self, image_id=None, base_region=None, regions=None):
         for region in regions:
             log.info('Copy %s to %s started' % (image_id, region))
             self.set_region(region)
-            self.image_name = get_image_name_from_ami_name(image.name, region)
```

Comment:
if we're going to remove this function call then we can remove the import of this function at the top of the file